src/site/content/en/blog/aurora-resource-inlining/index.md

```diff
@@ -0,0 +1,212 @@
+---
+title: 'Resource Inlining in JavaScript frameworks'
+subhead: Improving Largest Contentful Paint across the JavaScript ecosystem.
+authors:
+  - mgechev
+date: 2021-06-28
+hero: image/S838B7UEsdXmwrD8q5gvNlWTHHP2/yXASsFeUg39y0K7aFJIY.jpg
+alt: A labyrinth.
+description: Learn about the latest optimizations implemented in JavaScript frameworks in collaboration with project Aurora.
+tags:
+  - blog
+  - angular
+  - next.js
+  - nuxt.js
+  - cwv
+  - lcp
+---
+
+As part of project [Aurora](/introducing-aurora/), Google has been working with popular Web
```

Comment:
```suggestion
As part of project [Aurora](/introducing-aurora/), Google has been working with popular web
```

Reply:
@rachelandrew This came up in the Wasm post too, our styleguide prescribes not capitalizing "web": https://web.dev/handbook/word-list/